### PR TITLE
feat: #1009 serverless deployment for cloud apps

### PR DESCRIPTION
**Pull request by tanphamhaiduong**
Changes
- Clone serverless.yml for cloud apps

### CODE REVIEW

File: packages/aml-checklist/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+                HTTPPort: 80
+                HTTPSPort: 443
+                OriginProtocolPolicy: https-only
+              ## In case you want to restrict the bucket access use S3OriginConfig and remove CustomOriginConfig
+              # S3OriginConfig:
+              #   OriginAccessIdentity: origin-access-identity/cloudfront/E127EXAMPLE51Z
+          Enabled: 'true'
+          ## Uncomment the following section in case you are using a custom domain
+          # Aliases:
+          # - dev.smb-app.reapit.cloud
```
Comment (willmcvay) [inline packages/aml-checklist/serverless.yml:75]: Can we add the correct domains to the aliases for the benefit of infra team eg `aml-app.reapit.cloud` etc may need to use a yaml helper so we can add the `${self:provider.stage}` prefix if dev

File: packages/elements/serverless.yml [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+                HTTPPort: 80
+                HTTPSPort: 443
+                OriginProtocolPolicy: https-only
+              ## In case you want to restrict the bucket access use S3OriginConfig and remove CustomOriginConfig
+              # S3OriginConfig:
+              #   OriginAccessIdentity: origin-access-identity/cloudfront/E127EXAMPLE51Z
+          Enabled: 'true'
+          ## Uncomment the following section in case you are using a custom domain
+          # Aliases:
+          # - dev.smb-app.reapit.cloud
```
Comment (willmcvay) [inline packages/elements/serverless.yml:75]: See above on aliases

File: packages/demo-site/serverless.yml [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+                HTTPPort: 80
+                HTTPSPort: 443
+                OriginProtocolPolicy: https-only
+              ## In case you want to restrict the bucket access use S3OriginConfig and remove CustomOriginConfig
+              # S3OriginConfig:
+              #   OriginAccessIdentity: origin-access-identity/cloudfront/E127EXAMPLE51Z
+          Enabled: 'true'
+          ## Uncomment the following section in case you are using a custom domain
+          # Aliases:
+          # - dev.smb-app.reapit.cloud
```
Comment (willmcvay) [inline packages/demo-site/serverless.yml:75]: See above on aliases

File: packages/aml-checklist/serverless.yml (hunk 1 above)
Comment (tanphamhaiduong) [reply packages/aml-checklist/serverless.yml:75]: @willmcvay can, but I still let it comment because it'll have problem when deploy

Comment (willmcvay) [reply packages/aml-checklist/serverless.yml:75]: @tanphamhaiduong Sure, I get that - it's to help with the handover, I know it won't work at this stage!

**Pull request merged**